droopler.install

```diff
@@ -1450,8 +1450,16 @@ function droopler_update_8117() {
   /** @var Drupal\d_update\Updater $updater */
   $updater = \Drupal::service('d_update');
 
+  $configs = [
+    'd_media' => [
+      'image.style.large_gallery_thumbnail' => '',
+    ],
+  ];
+
   $values['droopler__droopler_update_8117_1'] = update_modules_configuration(['d_p_side_by_side'], '8117');
   $values['droopler__droopler_update_8117_2'] = update_modules_configuration(['d_product'], '8117');
+  $values['droopler__droopler_update_8117_3'] = $updater->importConfigs($configs);
+  $values['droopler__droopler_update_8117_4'] = update_modules_configuration(['d_blog'], '8117');
```

Comment:
te 3 linijki każda z wywolaniem `update_modules_configuration` na pojedynczym module, to trzbea do jednej zrefaktorowac.. tam przeciez podajemy arrayke :thinking: